README.md

```diff
@@ -85,6 +87,6 @@ Your output could be something like:
 The most common mood is :-)
 ```
 
-**think**: Should you use an array or a hash to solve this problem? Why?
+**think**: Should you use an array or a hash to solve this problem? Why? I used a hash because order doesn't matter (I think)- I'm concerned about being able to tie the count to a specific emoji and not a specific position.
```

Comment:
Ordering doesn't matter correct. And Hash is what I would have picked as well. But I my reasoning is different. I only care about emoticons and their counts, to me this is an obvious key/value pair relationship and as it turns out it's a usage for hashes in many whiteboarding problems (counting the occurances of something)